logproof/src/linear_relation.rs

```diff
@@ -333,6 +370,23 @@ where
 
         Self { s: s.clone(), vk }
     }
+
+    /// Currently we only really use this to pull out the first n bits, but we
```

Comment:
This isn't a very good doc comment :)

Reply:
Fixed to mention that the index is into S.